service/src/main/java/crawlercommons/urlfrontier/service/memory/MemoryFrontierService.java

```diff
@@ -27,13 +29,8 @@ public class MemoryFrontierService extends AbstractFrontierService {
     private static final org.slf4j.Logger LOG =
             LoggerFactory.getLogger(MemoryFrontierService.class);
 
-    public MemoryFrontierService(final Map<String, String> configuration, String host, int port) {
```

Comment:
I am pretty sure we need this constructor. any reason you want to remove it?

Reply:
Indeed, I can't find a valid reason to remove it...will revert